setup.py

```diff
@@ -94,9 +95,9 @@
         "ijson>=3.1.4,<4",
         "importlib-metadata",
         "ipython>=8.5.0,<9",
-        "packaging>=20.9,<21",
+        "packaging>=21,<22",
         "pandas>=1.3.0,<2",
-        "pluggy>=1.0.0,<2",
+        "pluggy>=0.12,<2",
```

Comment:
What is causing this conflict for you?

Reply:
pytest, without this you cant resolve to use pytest==7.0

Reply:
When I do:

```bash
pip list | grep -e pytest -e pluggy
```

I see both:

```
pluggy                        1.0.0
pytest                        7.2.1
```

So we should be ale to keep `1.0.0` as the minimum.
Are you sure you need `0.12` as the minimum?
Am I reading this diff correctly? You are wanting to change from `1.0.0` as the minimum to `0.12`?